src/components/user-block/react/UserBlock.tsx

```diff
@@ -0,0 +1,193 @@
+import React from 'react';
+
+import classNames from 'classnames';
+
+import { Theme, Themes } from 'LumX/components';
+
+import { COMPONENT_PREFIX } from 'LumX/core/react/constants';
+
+import { ThumbnailVariants } from 'LumX';
+import { Thumbnail } from 'LumX/components/thumbnail/react/Thumbnail';
+import { IGenericProps, getRootClassName } from 'LumX/core/react/utils';
+import { handleBasicClasses } from 'LumX/core/utils';
+
+/**
+ * Authorized size values.
+ */
+enum Sizes {
+    s = 's',
+    m = 'm',
+    l = 'l',
+}
+type Size = Sizes;
+
+/**
+ * Authorized Orientations.
+ */
+enum Orientations {
```

Comment:
This enum can be exported in a more global constants since `ImageBlock` will be using it

Reply:
Yep, you are right.
I created the `ORIENTATIONS` constant  in `LumX/core/react/constants`